sonarqube/content.md

```diff
@@ -8,6 +8,18 @@
 
 Here you'll find the Docker images for the Community Edition, Developer Edition, Enterprise Edition, and Data Center Edition of SonarQube.
 
+## LTS image
+
+SonarQube LTS is here to ensure stability, the LTS only get security patches and bugfixes, please [see here](https://www.sonarsource.com/products/sonarqube/downloads/lts/) to get detailed informations
+
+The `lts-<edition>` tag behave as the **latest** `lts-<edition>` tag, which means it will be updated each time a new LTS is released.
+
+So relying on the lts tag can have disruptive effects. We highly recommend using 9.9.x-<edition> tag instead of lts-<edition> (see [here](./README.md#supported-tags-and-respective-dockerfile-links) for the complete list of tags).
+
+The current LTS version is `9.9.0`.
+
+If you use the plain `lts` tag, it default to the community version (lts-community).
+
```

Comment:
Is there a specific reason why we want to highlight the LTS? It can be seen as an incentive to install the LTS. If the main purpose is to warn about the lts tag, we may just want to add a note about it. Maybe in the installation section?

Reply:
For ex, replicating the documentation:
The lts tag on Docker images is replaced with every new LTS release. If you want to avoid an automatic upgrade, we recommend using the corresponding 9.9.x???-<edition> tag instead of lts-<edition>.